src/main/java/com/google/devtools/build/lib/buildeventservice/client/BuildEventServiceGrpcClient.java

```diff
@@ -67,6 +81,8 @@ protected BuildEventServiceGrpcClient(
     this.besAsync = besAsync;
     this.besBlocking = besBlocking;
     this.channel = channel;
+    this.buildRequestId = "testing/" + UUID.randomUUID();
```

Comment:
nit: move the testing string to a constant in the class

Reply:
This constructor seems unused: https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/buildeventservice/client/BuildEventServiceGrpcClient.java;l=63;drc=ceac50cc97d35b6e29d2940c1f925940f92be0b6;bpv=1;bpt=1

If it's unused at Google, we should just remove it.